js/Source/Bootstrap.Popup/Delegator.BS.ShowPopup.js

```diff
@@ -19,14 +19,24 @@ provides: [Delegator.BS.ShowPopup]
 ...
 */
 
-Delegator.register('click', 'BS.showPopup', {
+(function(){
 
-  handler: function(event, link, api){
-    var target = api.get('target') ? link.getElement(api.get('target')) : document.id(link.get('href').split("#")[1]);
-    event.preventDefault();
-    if (!target) api.fail('Could not find target element to activate: ' + (api.get('target') || link.get('href')));
-    api.getBehavior().apply(target);
-    target.getBehaviorResult('BS.Popup').show();
-  }
+  var triggers = {};
 
-});
\ No newline at end of file
+  ['show', 'hide'].each(function(action){
+
+    triggers['BS.'+ action + 'Popup'] = {
+      handler: function(event, link, api){
+        var target = api.get('target') ? link.getElement(api.get('target')) : document.id(link.get('href').split("#")[1]);
+        event.preventDefault();
+        if (!target) api.fail('Could not find target element to activate: ' + (api.get('target') || link.get('href')));
+        api.getBehavior().apply(target);
+        target.getBehaviorResult('BS.Popup')[action]();
+      }
+    };
+
+  });
+
+  Delegator.register('click', triggers);
+
+})();
```

Comment:
Please update the file name to be `Delegator.showHidePopup`; add to the provisions that it now provides `Delegator.hidePopup`, update the docs file - rename the file, update the title of the page, add an example of hiding.